src/index-node.ts

```diff
@@ -28,9 +28,21 @@ export class SiweClient extends SiweImpl {
 
   async _extractCookies(headers?: Headers): Promise<void> {
     const cookieRecord: Record<string, string> = {}
-    const headerSetCookies = (headers as any).raw()[
-      'set-cookie'
-    ] as Array<string>
+    // previously
+
+    let headerSetCookies: Array<string> = []
+
+    try {
+      // returns headers.raw is not a function while running the sdk on a repl
+      headerSetCookies = (headers as any)?.raw()['set-cookie'] as Array<string>
+    } catch {
+      //  returns headers.get is not a function while running test cases
```

Comment:
that might be because our mocks are incomplete. in any case, there should be a test that enters this block to make sure it works as expected